qcengine/util.py

```diff
@@ -412,7 +433,7 @@ def scratch_directory(child: str = None, parent: str = None, messy: bool = False
 
 
 @contextmanager
-def disk_files(infiles: Dict[str, str], outfiles: Dict[str, None], cwd: Optional[str] = None) -> Dict[str, str]:
+def disk_files(infiles: Dict[str, Union[str, bytes]], outfiles: Dict[str, None], cwd: Optional[str] = None, as_binary: Optional[List[str]] = None) -> Dict[str, Union[str, bytes]]:
```

Comment:
Can we  do `disk_files(infiles, outfiles, *, ...)`. I a bit worried about mixing up the order after looking at the signature in execute.

Reply:
sure. I actually added them, then deleted as over-involved for fn not called directly. will re-add.